config/default.yml

```
@@ -103,7 +103,17 @@ Minitest/GlobalExpectations:
   Description: 'This cop checks for deprecated global expectations.'
   StyleGuide: 'https://minitest.rubystyle.guide#global-expectations'
   Enabled: true
+  EnforcedStyle: any
+  Include:
+    - '**/spec/**/*'
+    - '**/*_spec.rb'
```

Can you update to the following?

```suggestion
    - '**/test/**/*'
    - '**/*_test.rb'
    - '**/spec/**/*'
    - '**/*_spec.rb'
```

---

I noticed that the values are merged with the global setting, but this makes it more explicit and avoids any merge configuration override.